src/java.base/share/conf/security/java.security

```diff
@@ -28,6 +28,46 @@
 # Properties in this file are typically parsed only once. If any of the
 # properties are modified, applications should be restarted to ensure the
 # changes are properly reflected.
+#
+# The special "include" property can be defined one or multiple times in
+# this file with a filesystem path value. The effect of each definition
+# is to include a referred security properties file inline, adding all
+# its properties. Security properties defined before an include statement
+# may be overridden by properties in the included file, if their names
+# match. Conversely, properties defined after an include statement may
+# override properties in the included file.
+#
+# Included files, as well as files pointed by java.security.properties,
```

Comment:
Should "pointed" be "pointed to"?

Reply:
Hmm, you're probably right. I'll make the change.